### PR TITLE
Redmine

### PR DESCRIPTION
**Pull request by ashalya**

### CODE REVIEW

File: redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		<groupId>org.wso2.carbon</groupId>
 		<artifactId>esb-connector-parent</artifactId>
 		<version>4.3.0-SNAPSHOT</version>
-		<relativePath>../../../pom.xml</relativePath>
+		<relativePath>../../../../pom.xml</relativePath>
```
Comment (kesavany) [inline redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Add the license header.

Comment (ashalya) [inline redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: fixed in latest version

File: redmine/pom.xml [hunk 2]
```diff
@@ -1,190 +0,0 @@
...
-    <url>http://wso2.org</url>
-    <modules>
-    	<module>redmine</module>
-    </modules>
-
-	<dependencies>
-		<dependency>
-			<groupId>org.wso2.carbon</groupId>
-			<artifactId>org.wso2.carbon.mediation.initializer</artifactId>
-			<version>4.2.0</version>
```
Comment (msmshariq) [inline redmine/pom.xml]: Manage dependency versions as properties.

Comment (ashalya) [inline redmine/pom.xml]: Fixed in latest code. All dependency moved to parent pom

**Pull request merged**